mix/MixClient.cpp

```diff
@@ -44,27 +44,18 @@ namespace mix
 
 const Secret c_defaultUserAccountSecret = Secret("cb73d9408c4720e230387d956eb0f829d8a4dd2c1055f96257167e14e7169074");
 const u256 c_mixGenesisDifficulty = c_minimumDifficulty; //TODO: make it lower for Mix somehow
-
-class MixBlockChain: public dev::eth::BlockChain
+	
+bytes MixBlockChain::createGenesisBlock(h256 _stateRoot)
```

Comment:
update
The merge was correct. 
Besides that, MixBlockChain was not a part of this pull request. The only change that I made, was moving Class definition to .h file. Also, it is not code duplication. MixBlockChain `createGenesisBlock` is completely different.